pkg/mocks/client.go

```diff
@@ -55,6 +64,22 @@ func NewClient(objects ...runtime.Object) client.Interface {
 	return newClient(makeVersionsList(v1beta1.SchemeGroupVersion), objects...)
 }
 
+func NewClientWithFake(objects ...runtime.Object) (client.Interface, *testing.Fake) {
+	return newClientWithFake(makeVersionsList(v1beta1.SchemeGroupVersion), objects...)
+}
+
 func NewClient1_4(objects ...runtime.Object) client.Interface {
 	return newClient(makeVersionsList(v1alpha1.SchemeGroupVersion), objects...)
 }
+
+func normalizeName(name string) string {
```

Comment:
it is formatNormalize name (not exported) and doesn't handle "/" symbol (and should not do it). Considering that this is a very simple method of only few lines of code it seems that having separate methods is a more natural solution